manifests/stable/cluster-kube-descheduler-operator.v4.14.0.clusterserviceversion.yaml

```diff
@@ -28,9 +28,9 @@ metadata:
         }
       ]
     certified: "false"
-    containerImage: quay.io/openshift/origin-cluster-kube-descheduler-operator:4.13
+    containerImage: quay.io/openshift/origin-cluster-kube-descheduler-operator:4.14
     createdAt: 2021/10/13
-    olm.skipRange: ">=4.10.0 <4.13.0"
+    olm.skipRange: ">=4.10.0 <4.14.0"
```

Comment:
4.10.0 -> 4.11.0 (we always support the latest 3 versions only)

Reply:
With https://github.com/openshift/cluster-kube-descheduler-operator/blob/master/manifests/art.yaml#L10-L11 bumpding 4.10.0 -> 4.11.0 as well